src/js/apps/admin/program/workflows/workflows_app.js

```diff
@@ -20,6 +23,32 @@ export default App.extend({
     this.actions = actions;
 
     this.showChildView('content', new ListView({ collection: actions }));
+
+    if (!_DEVELOP_) return;
+
+    const actionDroplistMenu = new Backbone.Collection([
+      {
+        onSelect: () => {
+          Radio.trigger('event-router', 'program:action:new', this.program.id);
+        },
+        id: 'new-action',
```

Comment:
bah I just saw one other thing worth doing.  We're not using these `id`s so might as well remove them.  Also instead of using `name` let's just use `text` and remove `attr: 'name'` from the component definition.

Reply:
no problem